packages/bind-mount/package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "@prairielearn/bind-mount",
-  "version": "1.0.7",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "version": "1.1.0",
```

Comment:
Don't manually bump the version, this will be done automatically by [`changesets`](https://github.com/changesets/changesets) on master.

Reply:
Fixed in 2d05eadfd13f33929fd01a2cdf58339090bfa062.